### DISCUSSION TITLE
Why source strings in are empty? (source text is stored in Key)

### DISCUSSION THREAD

**Discussion by pktiuk** [WeblateOrg/weblate#7793] [Q&A] [answered]
[I have a problem](https://github.com/AntiMicroX/antimicrox/issues/1#issuecomment-1160118985), because in my project there are lot of strings which are not translatable, because they are marked as `Source in Review`.

![image](https://user-images.githubusercontent.com/45544416/175032113-f21fec06-5426-468c-9a4c-ed557ab2e8ae.png)

It appears to be caused by untranslated source strings

![image](https://user-images.githubusercontent.com/45544416/175032256-4543e8c6-d605-4b04-8df4-58f3689a3b55.png)

This example comes from `.ui` file generated by QT and it looks like this

```xml
     <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Input to calibrate:</string>
           </property>
          </widget>
         </item>
         <item>
```
I don't understand, why these source strings are not translated by default?  
How can I deal with this problem without manually editing all of these source strings?

**Comment by pktiuk** [comment]
I think it may be caused by unfinished state in translation ( `.ts`) file.  

```xml
    <message>
        <location filename="../../../src/gui/calibration.ui" line="70"/>
        <source>Input to calibrate:</source>
        <translation type="unfinished"></translation>
    </message>
```

Is there any simple way of marking an empty string as translated one?

**Comment by nijel** [comment, marked as answer]
I guess your translation is configured as monolingual, while the files are bilingual
